src/javascript/app/pages/trade/markets.jsx

```diff
@@ -79,8 +82,21 @@ class Markets extends React.Component {
             const submarket = Object.keys(this.markets[market_symbol].submarkets).sort(sortSubmarket)[0];
             underlying_symbol = Object.keys(this.markets[market_symbol].submarkets[submarket].symbols).sort()[0];
         }
+        const is_synthetic = symbol => /^(synthetic)/i.test(symbol);
+        const is_uk = State.getResponse('authorize.country') === 'gb';
+        const is_malta = State.getResponse('landing_company.gaming_company.shortcode') === 'malta';
```

Comment:
let's change the variable name to `is_mlt` instead of `is_malta` to be more consistent! 🙏🏼

Reply:
sure